ingest/test/lora_python_report_submit/lora.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package helium.poc_lora;
+package helium.poc_iot;
```

Comment:
looks like this file needs a rename as well as long as we're keeping it around

Reply:
actually i will remove this.  Its no longer relevant

Reply:
fixed here: 2d86cc6e0b517f6f300e1eb39df18d2891ebfb35